apps/codelab/src/app/codelabs/codelabs-routing.module.ts

```diff
@@ -19,11 +20,20 @@ const routes: Routes = [
   },
   {
     path: 'admin',
-    loadChildren: () => import('../modules/admin/admin.module').then(m => m.AdminModule)
+    loadChildren: () => import('./admin/admin.module').then(m => m.AdminModule)
   },
   {
     path: 'sync',
-    loadChildren: () => import('../modules/sync/sync.module').then(m => m.SyncAdminModule)
+    loadChildren: () => import('./sync/sync.module').then(m => m.SyncAdminModule)
+  },
+  {
+    path: 'typescript',
```

Comment:
Could you move it to angular-routing module? Inside of angular-routing it should be the first item of Routes and redirect to 'typescript' not 'angular/typescript'